apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -158,32 +159,31 @@ public static void main(String[] args) {
         new Configuration[] {
           new WebInfConfiguration(),
           new WebXmlConfiguration(),
+          new WebAppConfiguration(),
           new MetaInfConfiguration(),
           new FragmentConfiguration(),
           new JettyWebXmlConfiguration(),
           new AnnotationConfiguration()
         });
 
     // Allow webapps to see but not override SLF4J (prevents LinkageErrors).
-    webapp.getSystemClasspathPattern().add("org.slf4j.");
-
+    webapp.addSystemClassMatcher(new ClassMatcher("org.slf4j."));
```

Comment:
I was never sure that this was actually needed or whether it did the same thing at the above. I meant to get back to research it but have not yet.

Reply:
I looked back at the documentation and tested different possible configurations. What we have here technically works, but I rewrote it in a similar way to how it's done in Jetty 9 (https://www.eclipse.org/jetty/javadoc/jetty-9/org/eclipse/jetty/webapp/WebAppContext.html). Avoids using the new classMatcher() constructor.